<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointerSettingsWidget</class>
 <widget class="QWidget" name="PointerSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>134</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="square_sel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../../Icons.qrc">
     <normaloff>:/icons/assets/icons/selections/square_sel.png</normaloff>:/icons/assets/icons/selections/square_sel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxStroke">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Stroke</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxFill">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
  </widget>
  <widget class="QToolButton" name="circle_sel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../../Icons.qrc">
     <normaloff>:/icons/assets/icons/selections/circle_sel.png</normaloff>:/icons/assets/icons/selections/circle_sel.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="strokeWidth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stroke Width</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
